roles/edpm_nova/tasks/install.yml

```diff
@@ -1,5 +1,25 @@
 ---
 
+- name: Create a directory for container health checks
+  ansible.builtin.file:
+    path: /var/lib/containers/healthchecks
+    state: directory
+    setype: container_file_t
+    owner: root
+    group: root
+    mode: '0755'
+  become: true
```

Comment:
if we wer to take thsi approch the permission on this directory are incorrect

the helthchecsk would not be in /var/lib/containers
they would be under /var/lib/openstack/<service>/ and should owned by the current user not root.

in the case of nova that directory is already create by this role with the correct permissions so this task is not required

Reply:
Ok, will move scripts there.